multitenancy.md

```diff
@@ -1,28 +1,36 @@
+
+Figure 1 shows the color-coded flow of two different tenants through Health Record Ingestion: 
+
+- The **red color** indicates Tenant 1's data. 
+- The **blue color** indicates Tenant 2's data. 
+- Data Integrator B is both red and blue, because it processes data for both tenants.
+
+**Figure 1: Core architecture, multi-tenancy and Health Record Ingestion**
 
----
 ![core-architecture](assets/img/multitenancy.png)
 
-**Note:** currently the Management API uses IBM Functions API keys for authentication, see [HRI Management API Keys](admin.md#hri-management-api-keys) for more details. 
+**Note:** Currently, the Management API uses IBM&reg; Functions API keys for authentication. For more information, see [Setting up Management API keys](admin.md#hri-management-api-keys). 
+
+#### About Event Streams topics
+In Event Streams, there must be at least one topic for every tenant and Data Integrator. To facilitate this, topics are named using the tenant and Data Integrator's name. Example:
+
+**ingest.tenant.data-integrator**
 
-This diagrams shows the flow of two different tenant's through the HRI via coloring. Red indicates Tenant 1's data and blue indicates Tenant 2's data. Data Integrator B is both red and blue, because it's processing data for both tenants.
+In Figure 1, Integrator B is processing data from two tenants and writes data to two topics, separating them by tenant. Event Streams credentials provided to Data Integrators are restricted to specific topics. For more information, see [Creating service credentials for Kafka permissions](admin.md#creating-service-credentials-for-kafka-permissions).
 
-#### Event Streams Topics
-In Event Streams, there must be at least one topic for every tenant and and Data Integrator. To facilitate this, topics are named using the tenant and Data Integrator's name, e.g. `ingest.tenant.data-integrator.*`. 
+##### About data types
+Health Record Ingestion is agnostic to the type of data being written to Kafka. In practice, a Data Integrator often provides a specific type of data, for example, claims data, clinical data, and imagery data, to Health Record Ingestion. Users and consumers of Health Record Ingestion might also want separate provided data by type. To do this, create additional topics and include another (data type) identifier at the end of the topic name before `.in`. Example:
 
-In the example above, Integrator B is processing data from two tenants and writes data to two topics, separating them by tenant. Event Streams credentials provided to Data Integrators are restricted to specific topics. See [Creating Service Credentials for Kafka Permissions](admin.md#creating-service-credentials-for-kafka-permissions) for more details.
+**ingest.t1.di1.claims.in**
```

Comment:
This looks like a section header, not an example. Either move it back inline with the paragraph or make a code block.